gossip/src/crds_value.rs

```diff
@@ -39,6 +41,9 @@ pub const MAX_VOTES: VoteIndex = 32;
 pub type EpochSlotsIndex = u8;
 pub const MAX_EPOCH_SLOTS: EpochSlotsIndex = 255;
 
+// This number is MAX_CRDS_OBJECT_SIZE - empty serialized RestartLastVotedForkSlots.
+const MAX_RESTART_LAST_VOTED_FORK_SLOTS_SPACE: usize = 825;
+
```

Comment:
Can you define this struct inside `RestartLastVotedForkSlots` struct?
similar to https://github.com/solana-labs/solana/blob/a851670d5/gossip/src/received_cache.rs#L20

Then you wouldn't need to add `RESTART_LAST_VOTED_FORK_SLOTS` in the name.

Reply:
Done.